app/helpers/application_helper.rb

```diff
@@ -38,6 +38,24 @@ def render_view_hooks(partial, locals: {}, feature: :addon)
     ;nil
   end
 
+  def collect_data_attrs_hooks(partial, locals: {}, feature: :addon)
```

Comment:
What's the advantage of going with this approach vs the existing `render_view_hooks`?

Reply:
If we have a `issues/send_to_bulk` partial that only contains the attr value and no html tags, something like:
```
<% if can? ... %>
   data-attr: attr-value
<% end %>
```

Couldn't we just use 
```
table_attributes.merge!(render_view_hooks('issues/send_to_bulk'))
```

Which would add the data-attr or nothing?